blockchain/common/common.go

```diff
@@ -74,7 +74,7 @@ type Params struct {
 	FluxMonitor json.RawMessage `json:"fluxmonitor"`
 
 	// Name FM:
-	FeedId    uint32 `json:"feed_id"`
+	FeedId    string `json:"feed_id"`
```

Comment:
Needed to check existence in job spec

Reply:
I'm not a big fan of making this a string, just to convert it to an int again. I think we could expect a type of `json.RawMessage` and check the size in bytes, and then unmarshal to int.

Reply:
Changed it to pointer